src/functionHelper.js

```diff
@@ -9,21 +10,76 @@ const uuid = require('uuid/v4');
 const handlerCache = {};
 const messageCallbacks = {};
 
+function runPythonHandler(funOptions, options) {
+    var spawn = require("child_process").spawn;
+    return function (event, context) {
+        var process = spawn('sls', ["invoke", "local", "-f", funOptions.funName],
+            { stdio: ['pipe', 'pipe', 'pipe'], shell: true, cwd: funOptions.servicePath });
+        process.stdin.write(JSON.stringify(event) + "\n");
+        process.stdin.end();
+        let results = ''
+        let hasDetectedJson = false;
+        process.stdout.on('data', (data) => {
+            let str = data.toString('utf8');
+            if (hasDetectedJson) {
+                // Assumes that all data after matching the start of the
+                // JSON result is the rest of the context result.
+                results = results + trimNewlines(str);
+            } else {
+                // Search for the start of the JSON result
+                // https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html#api-gateway-simple-proxy-for-lambda-output-format
+                const match = /{\n\s+"isBase64Encoded"|{\n\s+"statusCode"|{\n\s+"headers"|{\n\s+"body"/.exec(str);
+                if (match && match.index > -1) {
```

Comment:
if `match` is not falsy, will `match.index === -1`? I don't think the second check is necessary